### PR TITLE
fix(generation) - fix schema generation for enums in inheritance tree

### PR DESCRIPTION
**Pull request by pjpritch**

### CODE REVIEW

File: src/schema/TypeORMConverter.ts [hunk 1]
```diff
@@ -40,7 +40,16 @@ export function filenameToImportPath(filename: string): string {
 }
 
 export function extractEnumObject(column: ColumnMetadata): GraphQLEnumType {
```
Comment (BenBrostoff) [inline src/schema/TypeORMConverter.ts]: Could this whole thing just be:

```js
const modelEnums = column.entityMetadata.inheritanceTree.map(f => 
  getMetadataStorage()
  .getEnum(modelNames[f.name], column.propertyName)
);
return modelEnums.find(e => Boolean(e));
```

admittedly have not run this, just wondering if `for` loop with break can be removed here.

Comment (pjpritch) [reply src/schema/TypeORMConverter.ts]: my C / C++ was showing, wasn't it?

Comment (pjpritch) [reply src/schema/TypeORMConverter.ts]: your version was definitely prettier (and didn't include `break` or `goto`)

Comment (BenBrostoff) [reply src/schema/TypeORMConverter.ts]: Awesome :-)

File: src/decorators/EnumField.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { composeMethodDecorators, generatedFolderPath, MethodDecoratorFactory }
 
 interface EnumFieldOptions {
   nullable?: boolean;
-  default?: boolean;
+  default?: any;
```
Comment (deweyjose) [inline src/decorators/EnumField.ts]: +1

**Pull request merged**